kuma/static/js/wiki.js

```diff
@@ -306,19 +316,24 @@
     (function() {
         // don't run if no compat table on page with min 1 row
         var $compatTables = $('.bc-api table tbody tr');
+        var compatCSS, compatJS;
         if(!$compatTables.length) return;
 
         // don't run if waffle not active
         if(!win.waffle || !win.waffle.flag_is_active('compat_api')) return;
 
+        if(!win.mdn || !mdn.assets) return;
+        /* This will break if these assets are split into multiple files */
+        compatCSS = mdn.assets.css['wiki-compat-tables'][0];
+        compatJS = mdn.assets.js['wiki-compat-tables'][0];
```

Comment:
Awesome! Thanks for being clear about this. We could even make the `forEach` loop (above) more general to handle both the `syntax-prism` bundle and the `wiki-compat-tables` bundle, and any other variable-length bundles we want to dynamically inject in the future.